src/main/resources/static/api.yml

```diff
@@ -997,6 +1032,154 @@ paths:
           $ref: '#/components/responses/403Response'
         500:
           $ref: '#/components/responses/500Response'
+  /assessments:
+    get:
+      tags:
+        - Assessment data
+      summary: Gets assessments the user is authorised to view
+      parameters:
+        - in: query
+          name: filter
+          required: false
+          description: all (default) returns all assessments the user is authorised to view.  Allocated returns only those allocated to the user.
+          schema:
+            type: string
+            enum:
+              - all
+              - allocated
+      responses:
+        200:
+          description: successfully retrieved assessments
+          content:
+            'application/json':
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Assessment'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+  /assessments/{assessmentId}:
+    get:
+      tags:
+        - Assessment data
+      summary: Gets a single assessment by its id
+      parameters:
+        - in: path
+          name: assessmentId
+          required: true
+          description: Id of the assessment
+          schema:
+            type: string
+            format: uuid
+      responses:
+        200:
+          description: successfully retrieved assessment
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Assessment'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+    put:
+      tags:
+        - Assessment data
+      summary: Updates an assessment
+      parameters:
+        - in: path
+          name: assessmentId
+          required: true
+          description: Id of the assessment
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Updated assessment
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/UpdateAssessment'
+        required: true
+      responses:
+        200:
+          description: successfully retrieved assessment
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/Assessment'
+        401:
+          $ref: '#/components/responses/401Response'
+        403:
+          $ref: '#/components/responses/403Response'
+        500:
+          $ref: '#/components/responses/500Response'
+  /assessments/{assessmentId}/notes:
+    post:
```

Comment:
Given the user may be coming back and updating a note, then we might want to have a `PUT` action too?

Reply:
Actually, scratch that. For audit purposes, it's probably better to make these uneditable isn't it?

Reply:
yes, we didn't talk about it, but this design accommodates having multiple assessments without difficulty I think?

Reply:
Immutable would be easiest.  I suppose if we wanted to make them editable but auditable we could store a domain event each time one is edited?

Reply:
I think let's do immutable. This will make the model easy to reuse when we move to sending notes back and forth between AP staff and PPs